sections/main-product.liquid

```diff
@@ -450,6 +450,29 @@
   customElements.define('product-modal', ProductModal);
 {% endjavascript %}
 
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
+    function isIE() {
+      const ua = window.navigator.userAgent;
+      const msie = ua.indexOf('MSIE ');
+      const trident = ua.indexOf('Trident/');
+
+      return (msie > 0 || trident > 0);
+    }
+
+    if (!isIE()) return;
+    const hiddenInput = document.querySelector('product-form input[name="id"]');
```

Comment:
to be able to copy it over to feat. product I think i'd need to change this selector to be more specific to the section on which it's added. 

So i'm thinking:
```javascript
document.querySelector('#product-form-template-{{ section.id }} input[name="id"]');
```